bigtable-client-core-parent/bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/PutAdapterPerf.java

```diff
@@ -68,8 +72,8 @@ private static void putAdapterPerf(HBaseRequestAdapter adapter, Put put) {
     { 
       long start = System.nanoTime();
       for (int i = 0; i < count; i++) {
-        MutateRowRequest adapted = adapter.adapt(put);
-        BigtableGrpc.getMutateRowMethod().streamRequest(adapted);
+        RowMutation adapted = adapter.adapt(put);
+        BigtableGrpc.getMutateRowMethod().streamRequest(adapted.toProto(requestContext));
```

Comment:
This seems wrong for the perf class.  I'll work with you on updating this class in a different PR.

Reply:
As discussed, this change is ok